docs/contributing.md

```diff
@@ -1,65 +1,132 @@
-pipx uses [nox](https://pypi.org/project/nox/) for development, continuous integration testing, and automation.
+Thanks for your interest in contributing to pipx!
+
+
+## Dependencies
+pipx uses an automation tool called [nox](https://pypi.org/project/nox/) for development, continuous integration testing, and various tasks.
+
+`nox` defines tasks or "sessions" in `noxfile.py` which can be run with `nox -s SESSION_NAME`. Session names can be listed with `nox -l`.
+
+Install nox for pipx development:
+```
+pip install --user git+https://github.com/cs01/nox.git@5ea70723e9e6 nox
```

Comment:
Should we have a note here about equivalent `make develop`?  Is there a reason you dropped it?

Reply:
It felt kind of messy having `make` and `nox` together. I also wasn't sure I could assume windows users had `make installed`.